volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -972,7 +972,7 @@ def get_available_pages(self) -> Iterable[Tuple[int, int, int]]:
 
                 # If the entry is not a valid physical address then see if it is in transition.
                 elif mmpte.u.Trans.Transition == 1:
-                    physoffset = mmpte.u.Trans.PageFrameNumber << 12
+                    physoffset = (mmpte.u.Trans.PageFrameNumber &~ (0b1111 << 32)) << 12
```

Comment:
I'm not a big fan of using `~` for things (since python data types aren't bounded, so it ends up being represented by a negative number).  Is there a way to make this positive (such as `(mmpte.u.Trans.PageFrameNumber ^ (mmpte.u.Trans.PageFrameNumber & 0xf00000000)) << 12` or something)?